src/planwise/client/projects/handlers.cljs

```diff
@@ -22,8 +22,10 @@
 (register-handler
  :projects/invalidate-projects
  in-projects
- (fn [db [_]]
-   (update db :list asdf/invalidate!)))
+ (fn [db [_ new-projects]]
+   (if new-projects
+     (update db :list asdf/invalidate! into new-projects)
```

Comment:
This branch is not used. And I don't really like it. I'd rather invalidate-optimistically-update in-place.